project-base/app/tests/FrontendApiBundle/Functional/Customer/User/RegisterTest.php

```diff
@@ -133,4 +147,20 @@ public static function getRegisterQueryVariables(
             'country' => 'CZ',
         ];
     }
+
+    /**
+     * @param array $response
+     */
+    private function assertSuccessfulRegistration(array $response): void
+    {
+        $graphQlType = 'Register';
+        $this->assertResponseContainsArrayOfDataForGraphQlType($response, $graphQlType);
```

Comment:
this assertion is not quite necessary. the same does the `getResponseDataForGraphQlType` internally

Reply:
:+1: 94bd1a112a